src/test-app/app.js

```diff
@@ -653,11 +669,37 @@ export class TestApp {
         }
     }
 
+    cameraDataCapture(){
+      const camera = this._webGLScene.camera ;
+      return {
+        position: camera.position.toArray(),
+        rotation: camera.rotation.toArray(),
+        fov: camera.fov,
+        aspect: camera.aspect,
+        near: camera.near,
+        far: camera.far
+      };
+    }
+
+    restoreCameraData(cameraData) {
+      
+      var camera = new THREE.PerspectiveCamera();
+      camera.position.fromArray(cameraData.position);
+      camera.rotation.fromArray(cameraData.rotation);
+      camera.fov = cameraData.fov;
+      camera.aspect = cameraData.aspect;
+      camera.near = cameraData.near;
+      camera.far = cameraData.far;
+      this._webGLScene.camera = camera ;
```

Comment:
@enicolasgomez I tested this branch, and noticed that by re-assigning the camera it creates a bug while using snapshots. The canvas can't be rotated or zoom in, I think that's because we are reassigning the camera use